org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/DOMAttr.java

```diff
@@ -62,6 +62,11 @@ public DOMAttr getOwnerAttr() {
 			return DOMAttr.this;
 		}
 
+		@Override
+		public DOMNode getParentNode() {

```

Comment:
Why you did that? It doesn't follow the w3c spec https://www.w3.org/2003/01/dom2-javadoc/org/w3c/dom/Attr.html

`Thus, the Node attributes parentNode, previousSibling, and nextSibling have a null value for Attr objects.`

Reply:
This Node is not `Attr` though, this is a child of `Attr`. If the children of `Attr` are also not allowed to have a parent, then I will look into a different implementation of this.

Reply:
Ooops forget my comment, sorry.